kontrol-service/plugins/mock_github.go

```diff
@@ -5,17 +5,15 @@ var MockGitHub = map[string]map[string]string{
 	"https://github.com/h4ck3rk3y/a-test-plugin.git": {
 		"main.py": `REPLACED = "the-text-has-been-replaced"
 
-def create_flow(service_spec, deployment_spec, flow_uuid, text_to_replace):
-    deployment_spec['template']['metadata']['labels']['app'] = deployment_spec['template']['metadata']['labels']['app'].replace(text_to_replace, REPLACED)
-    deployment_spec['selector']['matchLabels']['app'] = deployment_spec['selector']['matchLabels']['app'].replace(text_to_replace, REPLACED)
-    deployment_spec['template']['spec']['containers'][0]['name'] = deployment_spec['template']['spec']['containers'][0]['name'].replace(text_to_replace, REPLACED)
+def create_flow(service_spec, pod_spec, flow_uuid, text_to_replace):
+    pod_spec['containers'][0]['name'] = pod_spec['containers'][0]['name'].replace(text_to_replace, REPLACED)
     
     config_map = {
         "original_text": text_to_replace
     }
     
     return {
-        "deployment_spec": deployment_spec,
+        "pod_spec": pod_spec,
```

Comment:
probably missing some context, but whats the reason for separating Service Spec and Deployment spec / changing this API? maybe added to the PR review would be helpful

Reply:
Added it to the PR description